Xamarin.Forms.Maps.Android/MapRenderer.cs

```diff
@@ -271,6 +271,11 @@ void PinOnPropertyChanged(object sender, PropertyChangedEventArgs e)
 			}
 		}
 
+		protected Marker GetMarkerForPin(Pin pin)
+		{
+			return _markers?.FirstOrDefault(m => m.Id == (string)pin.Id);
```

Comment:
While we're in here, we should remove this usage of `FirstOrDefault` and convert it to a `.Find`. It should be a bit better for performance.

Reply:
Cool, made me look up the SO thread on performance between both methods!

Done